pkg/collector/README.md

```diff
@@ -7,7 +7,4 @@ a check are all operations implemented in this package or one of the subpackages
 For further details, please refer to the specific READMEs:
 
 * [check](check/README.md)
-* [corechecks](corechecks/README.md)
-* [py](py/README.md)
-* [runner](runner/README.md)
```

Comment:
you don't have alternatives for these missing ones I guess

Reply:
Indeed there is just no README in those three directories...
I initially just set a link to the directory instead, but when I asked if they should be removed the answer was positive https://github.com/DataDog/datadog-agent/pull/21604#discussion_r1429810106